mobile/.bazelrc

```diff
@@ -14,7 +14,7 @@ build --define=admin_functionality=disabled
 build --define envoy_mobile_listener=disabled
 build --define=library_autolink=disabled
 build --define=envoy_mobile_swift_cxx_interop=enabled
-build --define=envoy_enable_http_datagrams=disabled
+build --define=enable_http_datagrams=disabled
```

Comment:
nit: instead of removing the "envoy_" prefix here, can we add it to this define elsewhere to be consistent?